docs/reference/distancesensor.md

```diff
@@ -4,12 +4,13 @@ Derived from [Device](device.md) and [Solid](solid.md).
 
 ```
 DistanceSensor {
-  MFVec3f  lookupTable   [ 0 0 0, 0.1 1000 0 ] # lookup table
-  SFString type          "generic"             # {"generic", "infra-red", "sonar", "laser"}
-  SFInt32  numberOfRays  1                     # [1, inf)
-  SFFloat  aperture      1.5708                # [0, 2*pi]
-  SFFloat  gaussianWidth 1                     # [0, inf)
-  SFFloat  resolution    -1                    # {-1, [0, inf)}
+  MFVec3f  lookupTable              [ 0 0 0, 0.1 1000 0 ] # lookup table
+  SFString type                     "generic"             # {"generic", "infra-red", "sonar", "laser"}
+  SFInt32  numberOfRays             1                     # [1, inf)
+  SFFloat  aperture                 1.5708                # [0, 2*pi]
+  SFFloat  gaussianWidth            1                     # [0, inf)
+  SFFloat  resolution               -1                    # {-1, [0, inf)}
+  SFFloat  redColorSensitivity      1                     # [0, 1]
```

Comment:
That seems more correct:
```suggestion
  SFFloat  redColorSensitivity      1                     # [0, inf)
```

Reply:
Actually even value bigger than 1 will work, it will increase the red color sensitivity of the sensor above the original Webots behavior. But we can go with 0-1, no problem. I even mention it in the WRL file. Maybe I should add a validation then?